scripted-tests/run/C_argc_argv-to-java_args/project/scala-native.sbt

```diff
@@ -1,7 +1,8 @@
 {
   val pluginVersion = System.getProperty("plugin.version")
-  if(pluginVersion == null)
-    throw new RuntimeException("""|The system property 'plugin.version' is not defined.
+  if (pluginVersion == null)
+    throw new RuntimeException(
+      """|The system property 'plugin.version' is not defined.
```

Comment:
`assumeStandardLibraryStripMargin = true` will fix this, but it seems to be non-idempotent in some corner cases.